### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by brianwchou**
## Description
I want to add more documentation to readme for ease of use. Here are some things I think should be added and or changed:

- name of project should be exactly the same as npm package
- include what min version of ganache this works with
- include detailed usages possibly in an examples folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
+});
+ ```
+
+## Other methods
+### advancing time
+```javascript
+helper.advanceTimeAndBlock(<seconds_to_advance_by>)
+```
+
+example is here https://github.com/ejwessel/TimeContract
```
Comment (ejwessel) [inline README.md]: maybe we should move this example into this repo, but exclude it from the npm publish

Comment (brianwchou) [reply README.md]: Yeah i was thinking about having a separate examples folder, with some usages of the library.

Comment (ejwessel) [reply README.md]: lets do it
make a new PR for it.

**Review by ejwessel — APPROVED** [review]
I think this is pretty good for now. It's at least much better than what it was before. 
If you have other things to add I'm up for it.

**Comment by brianwchou** [comment]
I like how it is currently. I think this is good to merge. We can always add more later

**Pull request merged**